### PR TITLE
Add support for the Person resource

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries

### CODE REVIEW

**Comment by remi-stripe** [comment]
@ob-stripe I never remember how to run the test suite locally? The `tox` commands never work for me unfortunately, do you know what to do?

**Comment by brandur-stripe** [comment]
@remi-stripe This is probably not a perfect system by any means, but using Pipenv usually works for me:

``` sh
pipenv install --dev
pipenv shell
tox -e py27
```

**Comment by remi-stripe** [comment]
@brandur-stripe yeah but ob once gave me the command to run a specific test and it was using pytest or something and I can never remember the right invocation lol

**Comment by jleclanche** [comment]
You can run specific tests matching by name with `-k`. eg `tox -e py36 -- -xk foo` will run tests on py36 env, -x will stop the test suite on the first error, and -k matches all tests that have `foo` in the name.

**Comment by remi-stripe** [comment]
@jleclanche Ah awesome I'll try that thank you!

File: tests/api_resources/test_account.py [hunk 1]
```diff
@@ -204,3 +214,66 @@ def test_is_creatable(self, request_mock):
             '/v1/accounts/%s/login_links' % TEST_RESOURCE_ID
         )
         assert isinstance(resource, stripe.LoginLink)
+
+
+class TestAccountPersons(object):
+    def test_is_creatable(self, request_mock):
+        resource = stripe.Account.create_person(
+            TEST_RESOURCE_ID,
+            dob=dict(
```
Comment (ob-stripe) [inline tests/api_resources/test_account.py]: Can you use the `{'day': 1, 'month': 1, 'year': 1980}` syntax instead?

**Comment by remi-stripe** [comment]
@ob-stripe PTAL

**Pull request merged**